### PR TITLE
rdar://problem/29689007 Updating the deploymet target for tests.

### PR DESCRIPTION
**Pull request by ematejska**
rdar://problem/29689007 Updating the deployment target for tests.

### CODE REVIEW

**Comment by ematejska** [comment]
@swift-ci please test

**Comment by slavapestov** [comment]
You have a typo in your commit message :)

Should this also go into master?

**Comment by ematejska** [comment]
@slavapestov Yes it should.

File: test/Misc/tbi.swift [hunk 1]
```diff
@@ -2,10 +2,6 @@
 // RUN:   -O -S %s -parse-as-library -parse-stdlib | \
 // RUN:   %FileCheck --check-prefix=TBI %s
 
-// RUN: %target-build-swift -target arm64-apple-ios7.0 -target-cpu cyclone \
-// RUN:     -O -S %s -parse-as-library -parse-stdlib | \
-// RUN:   %FileCheck --check-prefix=NO_TBI %s
```
Comment (jrose-apple) [inline test/Misc/tbi.swift]: This is okay for now, but I think it's still worth having *some* kind of reference platform to make sure the `and` instruction gets generated at all.

**Review by jrose-apple — APPROVED** [review]

File: test/Misc/tbi.swift (hunk 1 above)
Comment (jrose-apple) [inline test/Misc/tbi.swift]: But that can happen in the master branch version.

Comment (ematejska) [inline test/Misc/tbi.swift]: @jrose-apple  Yes, planning for that for master.

**Pull request merged**